packages/fs-router-vite-plugin/tests/__snapshots__/generator.spec.ts.snap

```diff
@@ -0,0 +1,182 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`generator works > should wire-up the routes for a "mdx" tree 1`] = `
```

Comment:
Is it possible to split this file into multiple cases? I wouldn't have a huge assertion file

Reply:
Would you prefer something like this https://vitest.dev/guide/snapshot.html#file-snapshots?

Reply:
Yeah. Every test case its own snap file

Reply:
Updated `fs-router-vite-plugin/tests/generator.spec.ts` to use `toMatchFileSnapshot`